mondoo.ps1

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env pwsh
+$fc = $host.UI.RawUI.ForegroundColor
+$host.UI.RawUI.ForegroundColor = "red"
+Write-Output "Mondoo command is deprecated. Use cnspec instead."
+$host.UI.RawUI.ForegroundColor = $fc
+cnspec $args
```

Comment:
For reference, here is an old CLI Wrapper Chris did: https://gitlab.com/mondoolabs/mondoo/-/blob/4.16.0/scripts/pkg/windows/msi/mondoo.cmd

Reply:
Does anything need to be done to execute mondoo.ps1 in place of mondoo.exe?  Like, an executable bit or something?

Reply:
i originally was using `Write-Host` which made coloring the text much simpler, but the powershell linter complained and suggested i should be using `Write-Output` instead.

i did run into some examples of previous wrappers that would add parameters (like the --config one in that example), but i thought we were going with an "assume nothing about locations of files" approach for this super-minimalist wrapper.

Reply:
> Does anything need to be done to execute mondoo.ps1 in place of mondoo.exe? Like, an executable bit or something?

It appears that `.ps1` is automatically handled in PowerShell.

```
PS C:\Users\hundr> ls c:\bin


    Directory: C:\bin


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         2/24/2023  12:09 PM      241344496 cnspec.exe
-a----         3/21/2023   3:07 PM            264 mondoo.ps1


PS C:\Users\hundr> echo $env:path
C:\Python311\Scripts\;C:\Python311\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\ProgramData\chocolatey\bin;C:\Program Files\Microsoft VS Code\bin;C:\Program Files\Git\cmd;C:\Program Files\Mondoo\;C:\Program Files\Go\bin;C:\Users\hundr\AppData\Local\Microsoft\WindowsApps;C:\bin;C:\Users\hundr\go\bin
PS C:\Users\hundr> mondoo help
mondoo : File C:\bin\mondoo.ps1 cannot be loaded because running scripts is disabled on this system. For more
information, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ mondoo help
+ ~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
PS C:\Users\hundr> mondoo help
Mondoo binary is deprecated. Use cnspec instead, Mondoo's cloud-native security scanner and CLI.
  ___ _ __  ___ _ __   ___  ___
 / __| '_ \/ __| '_ \ / _ \/ __|
| (__| | | \__ \ |_) |  __/ (__
 \___|_| |_|___/ .__/ \___|\___|
   mondoo™     |_|


cnspec is a cloud-native security testing tool for your entire fleet

Usage:
  cnspec [command]

Available Commands:
  bundle      Manage policy bundles
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  login       Register with Mondoo Platform
  logout      Log out from Mondoo Platform
  migrate     Migrate cnspec CLI configuration to the latest version
  run         Run an MQL query
  scan        Scan assets with one or more policies
  serve       Start cnspec in background mode
  shell       Interactive query shell for MQL
  status      Verify access to Mondoo Platform.
  vault       Manages vault environments
  version     Display the cnspec version.
  vuln        Scans a target for Vulnerabilities.

Flags:
      --config string      Set config file path (default $HOME/.config/mondoo/mondoo.yml)
  -h, --help               help for cnspec
      --log-level string   Set log level: error, warn, info, debug, trace (default "info")
  -v, --verbose            Enable verbose output

Use "cnspec [command] --help" for more information about a command.
PS C:\Users\hundr>
```